src/main/groovy/com/github/benmanes/gradle/versions/updates/DependencyUpdates.groovy

```diff
@@ -48,8 +49,9 @@ class DependencyUpdates {
 
   /** Evaluates the dependencies and returns a reporter. */
   DependencyUpdatesReporter run() {
-    Map<Project, Set<Configuration>> projectConfigs = project.allprojects.collectEntries { proj ->
-      [proj, proj.configurations.plus(proj.buildscript.configurations) as Set]
+    Map<Project, ConfigurationContainer> projectConfigs = project.allprojects.collectEntries { proj ->
+      proj.configurations.addAll(proj.buildscript.configurations)
```

Comment:
I think this could be the cause of `Cannot change dependencies of configuration ':classpath' after it has been resolved.` Try:
```groovy
def configurations = []
configurations.addAll(proj.configurations)
configurations.addAll(proj.buildscript.configurations)
[proj, configurations as Set]
```